### ISSUE TITLE
Broken links in array reference

### ISSUE THREAD

**Issue by pelikhan** [Microsoft/pxt#2760]
There is a number of broken references to the array docs. Also I've noticed that some docs (insertat) is missing the ``-`` between each word. 
```
checking docs
/types/array: broken link /blocks/arrays
/types/array: broken link /blocks/arrays/insert-at
/types/array: broken link /blocks/arrays
/reference/arrays: broken link /blocks/arrays/index-of
/reference/arrays: broken link /blocks/arrays/push
/reference/arrays: broken link /blocks/arrays/pop
/reference/arrays: broken link /blocks/arrays/shift
/reference/arrays: broken link /blocks/arrays/unshift
/reference/arrays: broken link /blocks/arrays/insert-at
/reference/arrays: broken link /blocks/arrays/remove-at
/reference/arrays: broken link /blocks/arrays/reverse
```

**Comment by ganicke** [comment]
This is due to a mismatch between master and v0. I'll bring the current docs to v0. The originals were stubs put in with array support.

**Comment by ganicke** [comment]
Fixed #2770, #2776.
